// saladbar-core/lib/index.js
import addClass from '../src/add-class';
import classList from '../src/class-list';
import dom from '../src/dom';
import domAll from '../src/dom-all';
import findParent from '../src/find-parent';
import getAttr from '../src/get-attr';
import getClass from '../src/get-class';
import getClasses from '../src/get-classes';
import getData from '../src/get-data';
import getProp from '../src/get-prop';
import getStyle from '../src/get-style';
import hasAttr from '../src/has-attr';
import hasClass from '../src/has-class';
import hasData from '../src/has-data';
import hasProp from '../src/has-prop';
import hasStyle from '../src/has-style';
import isAttr from '../src/is-attr';
import isData from '../src/is-data';
import isProp from '../src/is-prop';
import isStyle from '../src/is-style';
import removeAttr from '../src/remove-attr';
import removeClass from '../src/remove-class';
import replaceClass from '../src/replace-class';
import serialize from '../src/serialize';
import setAttr from '../src/set-attr';
import setData from '../src/set-data';
import setProp from '../src/set-prop';
import setStyle from '../src/set-style';
import toggleClass from '../src/toggle-class';

export {
  addClass,
  classList,
  dom,
  domAll,
  findParent,
  getAttr,
  getClass,
  getClasses,
  getData,
  getProp,
  getStyle,
  hasAttr,
  hasClass,
  hasData,
  hasProp,
  hasStyle,
  isAttr,
  isData,
  isProp,
  isStyle,
  removeAttr,
  removeClass,
  replaceClass,
  serialize,
  setAttr,
  setData,
  setProp,
  setStyle,
  toggleClass,
};
